src/Sylius/Bundle/PayumBundle/Resources/config/services.xml

```diff
@@ -79,5 +81,10 @@
         <!-- Dummy -->
         <service id="sylius.payum.dummy.action.capture_payment" class="%sylius.payum.dummy.action.capture_payment.class%" />
         <service id="sylius.payum.dummy.action.payment_status" class="%sylius.payum.dummy.action.payment_status.class%" />
+
+        <!-- Offline -->
+        <service id="sylius.payum.offline.action.capture_payment" class="%sylius.payum.offline.action.capture_payment.class%" public="false">
+            <tag name="payum.action" factory="offline" prepend="true" />
```

Comment:
if this https://github.com/Sylius/Sylius/pull/2641/files#r27721069 is done the prende true not needed any more.